monshan/overlook-solo
# Architecture/index and styling

# PR Overlook Solo Project
**Marika Shanahan**

### Fix or Feature?


### Summary of the changes

Builds the bare-bones framework of index.html and begins breaking out sass partials into separate files, some placeholder sass variables are defined and flex grid and flex box are applied where intended to best support future UXD implementation

### What ticket(s) do the changes resolve?

- Dashboard view (empty active area)
- Skeletons for future elements to be populated in bookings and available rooms sections

### Notes for the Reviewer

N/A

### How to Test Changes?

observe visible elements on localhost: 8080 or open the index.html file in the src directory
